plugins/modules/zos_copy.py

```diff
@@ -1375,129 +1208,178 @@ def copy_to_member(
             # *****************************************************************
             if "FSUM8976" in err and "EDC5091I" in err:
                 rc, out, err = self.run_command(
-                    "cp -X \"//'{0}'\" \"//'{1}'\"".format(new_src, dest)
+                    "cp -X \"//'{0}'\" \"//'{1}'\"".format(src, dest)
```

Comment:
I guess we could, that's the only system call I didn't change because browsing through ZOAU's documentation I had the impression that `-X` is still not supported.